docs/web/sdk-components-styling.mdx

```diff
@@ -247,13 +240,15 @@ import ListItemText from "@vertigis/web/ui/ListItemText";
 const ComponentWithIcon: FC<LayoutElementProperties<ComponentWithIconModel>> = (
     props
 ) => {
+    // highlight-next-line
+    const { iconId } = props.model;
```

Comment:
Since every component model has an `icon` property, we should not create a model that also has an `iconId` property, since that's a bit confusing. MenuItem has both, but that's for historic reasons.

Reply:
how about "idFromAppConfig" ?

Reply:
I was thinking you could just use `icon`? Like this:
```
const { icon } = props.model;
```

Reply:
Sounds good! Done.